### PR TITLE
Feature/add delete stations

### LINKED ISSUE #7: Add a form to add and remove stations

### PR DESCRIPTION
**Pull request by gardzielb**
Done

### CODE REVIEW

File: src/components/stations/AddStationDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+);
+
+export interface AddStationDialogProps {
+    setStations: (value: React.SetStateAction<Station[]>) => void;
+}
+
+const AddBikeDialog = (props: AddStationDialogProps) => {
+    const classes = useStyles()
+    const [open, setOpen] = React.useState(false);
+    const [name, setName] = React.useState("Unnamed station");
```
Comment (chlebnymaciej) [inline src/components/stations/AddStationDialog.tsx]: Please add type in state

File: src/components/stations/AddStationDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        setOpen(false);
+    };
+
+    const handleNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setName(event.target.value);
+    };
+
+    const handleSubmit = () => {
+        const station = addStation(name);
+        station.then(response => {
```
Comment (chlebnymaciej) [inline src/components/stations/AddStationDialog.tsx]: you can just to addStation(name).then(response => { ...

**Review by chlebnymaciej — CHANGES_REQUESTED** [review]
Think about it

File: src/components/stations/AddStationDialog.tsx (hunk 1 above)
Comment (gardzielb) [reply src/components/stations/AddStationDialog.tsx]: done

File: src/components/stations/AddStationDialog.tsx (hunk 2 above)
Comment (gardzielb) [reply src/components/stations/AddStationDialog.tsx]: done

**Review by chlebnymaciej — APPROVED** [review]
looks good now

**Pull request merged**